Source/GoBot/Contact+ViewDatabase.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Contact+ViewDatabase.swift
+//  Planetary
+//
+//  Created by Martin Dutra on 14/7/22.
+//  Copyright © 2022 Verse Communications Inc. All rights reserved.
+//
+
+import Foundation
+import SQLite
+
+extension Contact {
+    init?(row: Row, db: ViewDatabase) throws {
```

Comment:
I'm glad you like this pattern of having an initializer on the model with a database row. I wish there was a way we could have the compiler enforce that the Row contained all necessary columns, but I'm not sure if that's possible. It's certainly out of the scope of this PR if it is, I'm just wondering out loud.